src/internal/nativeModule.web.ts

```diff
@@ -70,12 +70,14 @@ declare global {
     webkitConnection?: Connection;
   }
 }
+const isWindowPresent = typeof window !== 'undefined';
```

Comment:
This does not affect approval - code looks great - just a thought for a future person who may have the
same questions I had?

```suggestion
// Use a constant test of this form because in SSR on next.js, optional chaining is not sufficient,
// but this test correctly detects that window is not available and allows for conditionals before access
const isWindowPresent = typeof window !== 'undefined';
```

Reply:
added explanation in comments